src/unstakeAg/index.ts

```diff
@@ -540,61 +549,93 @@ export class UnstakeAg {
       }),
     );
 
-    if (jup) {
-      const {
-        transactions: { setupTransaction, swapTransaction, cleanupTransaction },
-      } = await this.jupiter.exchange({
-        routeInfo: jup,
-        userPublicKey: user,
-        // since we're putting it in setup and cleanup always
-        wrapUnwrapSOL: false,
-        feeAccount: feeAccounts[WRAPPED_SOL_MINT.toString()],
-      });
-      if (setupTransaction) {
-        setupIxs.push(...setupTransaction.instructions);
+    // Legacy Transaction
+    if (asLegacyTransaction) {
+      if (jup) {
+        const { swapTransaction } = await this.jupiter.exchange({
+          routeInfo: jup,
+          userPublicKey: user,
+          wrapUnwrapSOL: true,
```

Comment:
```suggestion
          // since we're putting it in setup and cleanup always
          wrapUnwrapSOL: false,
```
? Not that it should matter since `filteredSwapIx` should filter it out below